01 - Getting Started.ipynb

```diff
@@ -24,13 +24,13 @@
     "### `conda` / `mamba`\n",
```

Comment:
<p>I found helpful to see in the video version, the sql that is run. Maybe add a cell with</p><pre class="ql-syntax" spellcheck="false">ibis.show_sql(
	penguins.filter(
&nbsp;&nbsp;&nbsp;		[
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;	penguins.island == "Torgersen",
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;	penguins.species == "Adelie",
&nbsp;&nbsp;&nbsp;		]
)) 
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>good idea!</p>